devito/operator/profiling.py

```diff
@@ -391,9 +397,41 @@ def timings(self):
         return OrderedDict([(k, v.time) for k, v in self.items()])
 
 
+def sniff_advisor_location():
+    """
+    Detect if Intel Advisor is installed on the machine and return
+    its location if it is.
+
+    """
+    try:
+        res = run(["advixe-cl", "--version"], stdout=PIPE, stderr=DEVNULL)
+        ver = res.stdout.decode("utf-8")
+        if not ver:
+            return None
+    except (UnicodeDecodeError, FileNotFoundError):
+        error("Intel Advisor cannot be found on your system, consider if you have sourced"
```

Comment:
`msg = "Intel Advisor ... sourced"`

so that you can do `error(msg)` instead of copy-pasting the whole string